python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -632,6 +633,52 @@ def dome_ready(az_motion: salobj.type_hints.BaseMsgType) -> bool:
                 f"Dome state: {MTDome.MotionState(az_motion.state).name}, inPosition: {az_motion.inPosition}"
             )
 
+    async def wait_for_dome_azEnabled(self, timeout: float) -> None:
```

Comment:
This method name should be `snake_case`, please rename it to `wait_for_dome_az_enabled`.

However, do you really need this method? won't the one you included for parking (`wait_for_dome_state`) do the work?

Reply:
azEnabled check removed